components/climate/thermostat.rst

```diff
@@ -285,17 +302,50 @@ Advanced Options
 **If configured, at least one of** ``default_target_temperature_low`` **and** ``default_target_temperature_high``
 **must be specified in the away mode configuration.**
 
-**Additional Actions/Behavior**
+Additional Actions/Behavior
+***************************
 
+- **target_temperature_change_action** (*Optional*, :ref:`Action <config-action>`): The action to call when the
+  thermostat's target temperature(s) is/are changed.
+- **startup_delay** (*Optional*, boolean): If set to ``true``, when ESPHome starts, ``min_cooling_off_time``,
+  ``min_fanning_off_time``, and ``min_heating_off_time`` must elapse before each respective action may be invoked.
+  This option provides a way to prevent damage to equipment (for example) disrupted by a power interruption.
+  Defaults to ``false``.
 - **fan_only_cooling** (*Optional*, boolean): If set to ``true``, when in the ``fan_only_mode`` climate mode,
   the ``fan_only_action`` will only be called when the observed temperature exceeds the upper set point plus
   ``cool_deadband``. When set to ``false`` (the default), ``fan_only_action`` is called immediately when
-  ``fan_only_mode`` is activated, regardless of the current temperature or set points.
-- **target_temperature_change_action** (*Optional*, :ref:`Action <config-action>`): The action to call when the
-  thermostat's target temperature(s) is/are changed.
-
-
-**Hysteresis Values**
+  ``fan_only_mode`` is activated, regardless of the current temperature or set points. Defaults to ``false``.
+- **fan_with_cooling** (*Optional*, boolean): If set to ``true``, ``fan_only_action`` will be called whenever
+  ``cool_action`` is called. This is useful for forced-air systems where the fan typically runs with cooling.
+  Defaults to ``false``.
+- **fan_with_heating** (*Optional*, boolean): If set to ``true``, ``fan_only_action`` will be called whenever
+  ``heat_action`` is called. This is useful for forced-air systems where the fan typically runs with heating.
+  Defaults to ``false``.
+- **max_cooling_run_time** (*Optional*, int > 0): Duration after which ``supplemental_cooling_action`` will be
```

Comment:
```suggestion
- **max_cooling_run_time** (*Optional*, :ref:`config-time`): Duration after which ``supplemental_cooling_action`` will be
```

Because this is a time value (and thus can/must have units)

Reply:
Perfect -- I'll update. Thank you yet again! 😄